### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by Dresmond**
Removed spaces following "(" and proceeding ")" in parameter lists.

### CODE REVIEW

**Comment by Dresmond** [comment]
Sending pull request to Matkat

File: main.cpp [hunk 1]
```diff
@@ -1,95 +1,252 @@
...
+};
+/* 
+10) ElevatorSystem UDT (embeds other structures, no primitives)
+
+/* 
+10) ElevatorSystem UDT (embeds other structures, no primitives)
+*/
+struct ElevatorSystem
+{
+    ElevatorCar car1;        // embedded structure
```
Comment (matkatmusic) [inline main.cpp:234]: add a couple member functions.  Something like `moveToFloor(int floorNum);` or `moveToFloor(ElevatorFloor floor);`

- [ ] 3) add a couple member functions.

File: main.cpp [hunk 2]
```diff
@@ -1,95 +1,252 @@
...
+    int previousFloor = 0;
+    int currentFloor = 0;
+    int nextFloor = 0;
+};
+/* 
+9) ElevatorCar UDT (embedded structure has embedded structures)
+*/
+struct ElevatorCar
+{ 
+    ElevatorDoors doors;    // embedded structure
```
Comment (matkatmusic) [inline main.cpp:222]: Add a member function that opens the doors. 

- [ ] 3) add a couple member functions.

**Review by matkatmusic — CHANGES_REQUESTED** [review]
2 things to implement.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**